controllers/operatorpolicy_controller.go

```diff
@@ -685,6 +704,24 @@ func (r *OperatorPolicyReconciler) handleSubscription(
 	return mergedSub, earlyConds, true, nil
 }
 
+// isApplicableConstraintsNotSatisfiable checks if the ConstraintsNotSatisfiable message includes the input
+// subscription or package. Some examples that it catches:
+// https://github.com/operator-framework/operator-lifecycle-manager/blob/dc0c564f62d526bae0467d53f439e1c91a17ed8a/pkg/controller/registry/resolver/resolver.go#L257-L267
+// - no operators found from catalog %s in namespace %s referenced by subscription %s
+// - no operators found in package %s in the catalog referenced by subscription %s
+// - no operators found in channel %s of package %s in the catalog referenced by subscription %s
+// - no operators found with name %s in channel %s of package %s in the catalog referenced by subscription %s
+func isApplicableConstraintsNotSatisfiable(subscriptionName, packageName, message string) (bool, error) {
```

Comment:
This function name is quite a mouthful. I would suggest `are` instead of `is`, to match the `Constraints` since that is plural, but that doesn't really make it feel better. What about something like `messageIncludesSubscription`?